app/api/probe_dock/metrics_api.rb

```diff
@@ -443,6 +443,20 @@ def current_project_version
           contribution
         end
       end
+
+      get :versionsWithNoResult do
+        authorize!(:organization, :data)
+
+        project_versions_rel = policy_scope(ProjectVersion).order('project_versions.created_at DESC')
+        project_versions_rel = project_versions_rel.joins(project: [:tests, :organization])
```

Comment:
I would avoid reassignments if there's no reason to do it and no useful comments in between. Simply write:

```
project_versions_rel = policy_scope(ProjectVersion)
  .order(...)
  .joins(...)
  .where(...)
```